src/library/arith.lisp

```diff
@@ -485,3 +504,118 @@
 (define-sxhash-hasher UFix)
 (define-sxhash-hasher Single-Float)
 (define-sxhash-hasher Double-Float)
+
+
+;;; `Quantization' 
+
+(coalton-toplevel
+  (define-instance (Quantizable Integer)
```

Comment:
FYI: I noticed the generated documentation for the Quantizable class didn't look good due to
mismatching autogenerated type var names and those that were mentioned in the docs.

Reply:
fixed